### DISCUSSION TITLE
ssl issue when host pocketbase in digitalocean droplet

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3566] [Q&A]
i use port 3000 for my frontend and a subdomain on 3080 using nginx reverse proxy and install ssl certificate (certbot)  but if i create a sub domain and trying to serve on the sub domain by using 
/lib/systemd/system/pocketbase.service
```
[Unit]
Description = pocketbase

[Service]
Type           = simple
User           = root
Group          = root
LimitNOFILE    = 4096
Restart        = always
RestartSec     = 5s
StandardOutput = append:/root/pb/errors.log
StandardError  = append:/root/pb/errors.log
ExecStart      = /root/pb/pocketbase serve sub.yourdomain.com

[Install]
WantedBy = multi-user.target  
```

than i facing an error to serve may be ssl conflict to each other..any solution to host pocketbase and my frontend and a subdomain in same droplet with ssl ?

**Comment by ganigeorgiev** [comment]
If you have a reverse-proxy (nginx) and manually manage the ssl certificate, then you don't have to specify a domain when starting the executable in the above `ExecStart` instruction.

To specify a listeneing address and port you can use the `--http` flag, something like:

```
/root/pb/pocketbase serve --http="127.0.0.1:3080"
```
